tests/integration/reportinglib/reporting_test.sh.in

```diff
@@ -1,6 +1,7 @@
 #! /bin/sh 
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:@CMAKE_BINARY_DIR@/lib:@reportinglib_LIB_DIR@
+export DYLD_LIBRARY_PATH=$LD_LIBRARY_PATH:@CMAKE_BINARY_DIR@/lib:@reportinglib_LIB_DIR@
```

Comment:
Since we now link with reportinglib with `libnrncorenrnmech` we can get rid of those, right? I tested on BB5 and on my mac and it seems to work fine.

Reply:
Yes!